vizro-core/src/vizro/static/css/layout.css

```
@@ -206,12 +206,36 @@ Current logic:
   }
 
   #page-header,
-  #left-side,
-  #right-side {
+  #right-side,
+  #left-side {
     zoom: 80%;
   }
 }
 
+@media screen and (width <= 576px) and (orientation: portrait) {
+  body:has(#collapsible-left-side.show) {
+    overflow: hidden;
+  }
+
+  #collapsible-left-side.show ~ .collapse-icon-div #collapse-icon {
+    right: 2px;
+    top: 70px;
+    z-index: 10;
```

I think I'd do this, so it doesn't jump when collapsing and uncollapsing. WDYT?
```suggestion
    right: -10px;
    z-index: 10;
```